modules/apps/export-import/export-import-report-service/src/main/java/com/liferay/exportimport/report/internal/exception/handler/ExportImportBatchEngineImportTaskExceptionHandler.java

```diff
@@ -116,6 +97,31 @@ private String _getExternalReferenceCode(Object item) {
 		}
 	}
 
+	private String _getModelName(Object item, String taskItemDelegateName) {
+		if (!Validator.isBlank(taskItemDelegateName)) {
+			if (taskItemDelegateName.startsWith("C_")) {
```

Comment:
Based on the code of method `ExportImportReportEntryUtil.getModelName`, I think you could update the current code there:

from
```java
	public static String getModelName(Object object) {
		[...]
		else if (object instanceof ObjectEntry objectEntry) {
		[...]
		
			if (objectDefinition != null) {
				return objectDefinition.getName();
			}
		}
```

to
```java
	public static String getModelName(Object object) {
		[...]
		else if (object instanceof ObjectEntry objectEntry) {
		[...]
		
			if (objectDefinition != null) {
				return objectDefinition.getShortName();
			}
		}
```